about.html

```diff
@@ -0,0 +1,81 @@
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li class="menuBorder"><a class="primary" href="#">REL Campaign</a></li>
+        </ul>
+    </nav>
+    <section>
+        <h3 class="welcomeTextAbout primary textCenter">Hello Gamers!</h3>
+        <h1 class="primary textCenter fontSize36 aboutTitle">ROMANIAN ESPORTS LEAGUE 2022</h1>
+        <p class="aboutDescription lineHeight15">The Romanian Esports League 2022 brings togather a community of professional players and enthusiasts in order to find the best players in a live tournament and have fun at the same time</p>
+        <p class="fontSize12 lineHeight15 textCenter">Please contact us per Email for any further questions<br> about REL 2022 at <strong>REL@esport.com</strong></p>
+    </section>
+    <section class="aboutLogo">
+        <p class="textCenter fontSize24 aboutLogoTitle lineHeight15 mLeft20 mRight20">The Romanian Esports League 2022 Logo</p>
+        <p class="aboutLogoTitleCompetition fontSize12 textCenter lineHeight15">The logo of The Romanian Esports League 2022 was decided through the logo competition from 8.June to 7.July.</p>
+        <div class="logoImg">
+            <img src="./img/RSL.png" alt="logo">
+        </div> 
+    </section>
+    <section class="history">
+        <p class="textCenter fontSize24 aboutLogoTitle lineHeight15">See the past REL competitons</p>
+        <p class="fontSize14 fontWeight700 textCenter mLeft20 mRight20">Take a look at the last two global summits which took place in Bucharest and Brasov</p>
+        <div class="flex flexWrap asa"> 
+            <button type="button" class="buc"> 
+                    <span class="white cityName">Bucharest</span>
+                    <br> <span class="white cityNameText">2021</span> 
+            </button>
+            <button type="button" class="bra"> 
+                <span class="white cityName">Brasov</span>
+                <br> <span class="white cityNameText">2020</span> 
+            </button>
+        </div> 
+    </section>
+    <section class="backgroundGray">
+        <p class="textCenter fontSize24 lineHeight15 partners white">Partners</p>
+        <ul class="flex flexWrap justifyCenter listNone">
+            <li class="sponsor fontSize20 fontWeight700 white ">Google</li>
+            <li class="sponsor fontSize20 fontWeight700 white">Microsoft</li>
+            <li class="sponsor fontSize20 fontWeight700 white">Facebook</li>
+            <li class="sponsor fontSize20 fontWeight700 white">Audi</li>
+            <li class="sponsor fontSize20 fontWeight700 white">Asus</li>
+        </ul>
+    </section>
+    <section class="grid2">
+        <img src="./img/RSL.png" alt="logo">
+        <div class="flex flexAlignCenter">Romanian Esports League 2022<br> All right reserved</div>
+    </section>
```

Comment:
- [ ] Kindly make sure you use a `footer` tag instead of a `section` tag 🙏 . This would make your webpage more accessible